packages/api/src/external/fhir/document/index.ts

```diff
@@ -144,3 +156,110 @@ export function idToFHIR(id: DocumentIdentifier): Identifier {
     use: id.use === "unspecified" ? undefined : id.use,
   };
 }
+
+/**
+ * Takes a CW resource and converts it to a FHIR resource
+ *
+ * @param resource
+ * @returns
+ */
+function convertToFHIRResource(resource: Contained): Resource | undefined {
+  if (resource.resourceType === ResourceType.Patient && resource.id) {
+    return {
+      resourceType: ResourceType.Patient,
+      id: resource.id,
+    };
+  } else if (resource.resourceType === ResourceType.Organization && resource.name) {
+    return {
+      resourceType: ResourceType.Organization,
+      name: convertCWNameToString(resource.name),
+    };
+  } else if (resource.resourceType === ResourceType.Practitioner && resource.name) {
+    return {
+      resourceType: ResourceType.Practitioner,
+      name: convertCWNameToHumanName(resource.name),
+    };
+  }
```

Comment:
Maybe also include `RelatedPerson` and `Person`?

Can we also add a Sentry message so we get notified when there's a contained resource we're not currently supporting, please?

Reply:
I'll add the sentry 👍 

Would be very helpful to see some payloads including `RelatedPerson` and `Person` being used, just if you have that at the ready. If not, I'll figure it out without it.

Reply:
I don't have those. Maybe we just leave the Sentry/Slack notif and we can redownload those after a patch if they come along.